servers/rendering/renderer_viewport.cpp

```diff
@@ -271,10 +271,12 @@ void RendererViewport::_draw_3d(Viewport *p_viewport) {
 #ifndef _3D_DISABLED
 	RENDER_TIMESTAMP("> Render 3D Scene");
 
-	Ref<XRInterface> xr_interface;
+	Ref<RefCounted> xr_interface;
```

Comment:
This should keep the `XRInterface` type most likely. `rendering_method.h` is included in `renderer_viewport.cpp` so it should have the define.